documentation/asciidoc/accessories/camera/libcamera_apps_post_processing_tflite.adoc

```diff
@@ -0,0 +1,48 @@
+=== Post-Processing with TensorFlow Lite
+
+These stages requires TensorFlow Lite (TFLite) libraries to be installed exporting the C++ API. Unfortunately the TFLite libraries are not normally distributed conveniently in this form, however, one place where they can be downloaded is https://lindevs.com/install-precompiled-tensorflow-lite-on-raspberry-pi/[here]. Please follow the installation instruction given on that page.
+
+==== `object_classify_tf` stage
+
+`object_classify_tf` uses a Google MobileNet v1 model to classify objects in the camera image. It can be obtained from https://storage.googleapis.com/download.tensorflow.org/models/mobilenet_v1_2018_08_02/mobilenet_v1_1.0_224_quant.tgz[], which will need to be uncompressed. You will also need the `labels.txt` file which can be found in https://storage.googleapis.com/download.tensorflow.org/models/mobilenet_v1_1.0_224_frozen.tgz[].
```

Comment:
Just as an FYI, appending an empty `[]` to the end of a URL is [optional](https://docs.asciidoctor.org/asciidoc/latest/macros/url-macro/) in AsciiDoctor.